app/templates/index.hbs

```diff
@@ -1 +1,7 @@
+{{delivery-breadcrumbs}}
+<h2>Data Delivery</h2>
+<p>Welcome to Data Delivery. This application is a project of <a href="https://genome.duke.edu">Duke GCB</a>.
+  Data Delivery and is designed to facilitate transactional delivery of data sets stored in in <a href="https://dataservice.duke.edu">Duke Data Service</a>.</p>
+<p>You can use this application to list and download your {{#link-to 'duke-ds-projects'}}DukeDS Projects{{/link-to}}, or Create, Send, and Manage {{#link-to 'deliveries'}}Deliveries{{/link-to}}
```

Comment:
It wasn't immediately obvious to me that the last two links are not just informational like the previous two - they take you to where you view projects and manage deliveries.

Reply:
Thanks for the feedback. This page is a first draft - something to stimulate feedback on Monday :)